src/imageio/format/avif.c

```diff
@@ -690,11 +698,6 @@ const char *name()
   return _("AVIF (8/10/12-bit)");
 }
 
-int flags(struct dt_imageio_module_data_t *data)
```

Comment:
IIUC, this will be needed when exiv2 supports writing XMP on AVIF (as OpenEXR). So maybe instead of removing add a clear comment about what to do when exiv2 is ok and maybe add the version information if already implemented in exiv2 even if not in a current release? (and of course comment out this code).

What do you think?

Reply:
Sure, can do that for future reference; will just return 0 and add the comment.